src/app.rs

```diff
@@ -78,6 +77,15 @@ impl App {
             config.marker_array_topics,
             config.map_topics,
         );
+        let base_link_pose = tf_listener
```

Comment:
One Big Question:
Why did you add this to the app?
I think it would be nice to have the math and logic in the initial pose publisher (maybe rename it) and offer a draw method to the App.
So we can keep the logic separate from the frontend.

Reply:
Wait do you mean specifically why do I lookup a tf here? If yes it's to cache the transformation, since we actually use it multiple times below